modules/hosted-control-planes-overview.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * architecture/control-plane.adoc
+
+
+:_content-type: CONCEPT
+[id="hosted-control-planes-overview_{context}"]
+= Hosted control planes overview (technology preview)
+
+Hosted control planes for {product-title} provides a way to decouple your control plane from your worker nodes so that you can work in a hybrid cloud approach. You can enable hosted control planes through the multicluster engine for Kubernetes operator version 2.0. In that version, only Amazon Web Services (AWS) hosted clusters are available.
```

Comment:
Hosted control planes helps Red Hat OpenShift customers focus on applications by reducing management overhead, optimizing cluster deployment time, and enabling hard separation of concerns between management and workloads.

 Hosted control planes is available on Amazon Web Services (AWS) as a [technology preview](https://docs.openshift.com/container-platform/4.10/rest_api/understanding-compatibility-guidelines.html#api-compatibility-common-terminology-dev-preview_compatibility-guidelines) feature enabled through the [multicluster engine for Kubernetes operator](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html/multicluster_engine/requirements-and-recommendations) version 2.0.